content/en/functions/strings.Count.md

```diff
@@ -0,0 +1,30 @@
+---
+title: strings.Count
+description: Returns the number of non-overlapping instances of a substring within a string.
+godocref:
+date: 2020-09-07
+publishdate: 2020-09-07
+lastmod: 2020-09-07
+categories: [functions]
+menu:
+  docs:
+    parent: "functions"
+keywords: [count, counting, character count]
+signature: ["strings.Count SUBSTR STRING"]
+workson: []
+hugoversion:
+relatedfuncs: []
+deprecated: false
+aliases: []
+---
+
+{{< new-in "0.74.0" >}}
+
+If `SUBSTR` is an empty string, this function returns 1 plus the number of Unicode code points in `STRING`.
+
+Example|Result
+:--|:--
+`{{ strings.Count "a" "aaabaab" }}`|5
```

Comment:
I think that when we go to great lengths to structure these functions to be "pipe friendly", we should also use that in examples, e.g.:

{{ "aaabaab" | strings.Count "a"  }}

Reply:
Done.